test/tests/concrete_ASR_swelling/asr_confined.i

```diff
@@ -1,14 +1,17 @@
 # @Requirement F3.50
```

Comment:
I believe this test to be a little bit too complex (it doesn't provide unit tests of the ASR model features). We can keep it for now, but I would prefer to have elementary tests in the future. I can do it once this has been merged.

Reply:
I'd like to keep this test for the time being just because I've verified that the new implementation exactly matches the old one. I agree completely that it would be best to have a set of tests targeted at specific aspects of the model, together with more complex tests like this that tie everything together, but let's do that next.